### ISSUE TITLE
2.0 - Settings connected green fails visual accessibility test

### ISSUE THREAD

**Issue by helveticade** [WhisperSystems/Signal-iOS#505]
![screen_shot_2015-01-25_at_10_31_07_am](https://cloud.githubusercontent.com/assets/737950/5893171/d60a52ec-a47d-11e4-815e-fa5101ddfe41.png)

Not enough visual contrast between background white and text colour.
